docs/usage/capabilities_list.md

```diff
@@ -0,0 +1,75 @@
+CapabilitiesList
+========================================
+
+Overview
+========================================
+
+**CapabilitiesList** is a Java application to list the capabilities
+of the "Mozilla-JSS" provider and other installed providers.
+It does it in two fashions brief and verbose. 
+
+It lets JSS contributors see what features the JDK implements (at different versions)
+and see what else we'd need to add in, to reach compatibility. Since we're 
+attempting to be a default crypto provider, it would be nice to ensure we're 
+mostly close to what the JDK provides. Hopefully other packages written to the
+JDK provider can have similar algorithms available under JSS and behave well. 
+Large gaps in coverage are likely an issue (such as #341 and #242).
+
+Usage
+========================================
+First build jss according to the instructions in jss/README.md
+You should see in the build directory tests_jss4.jar which is what
+contains the application along with the regular tests.
+From the jss/build directory execute
+./run_test org.mozilla.jss.tests.Capabilities
+
+and you should see a listings directory
+
+Running `tree listings` should produce something like
+
+listings/
+├── brief
+│   ├── Capabilities4JdkSASL.txt
+│   ├── Capabilities4Mozilla-JSS.txt
+│   ├── Capabilities4SunJCE.txt
+│   ├── Capabilities4SunJSSE.txt
+│   ├── Capabilities4SunPKCS11.txt
+│   ├── Capabilities4SunRsaSign.txt
+│   └── Capabilities4SUN.txt
+└── verbose
+    ├── Capabilities4JdkSASL.txt
+    ├── Capabilities4Mozilla-JSS.txt
+    ├── Capabilities4SunJCE.txt
+    ├── Capabilities4SunJSSE.txt
+    ├── Capabilities4SunPKCS11.txt
+    ├── Capabilities4SunRsaSign.txt
+    └── Capabilities4SUN.txt
```

Comment:
For things like code listings, it'd probably look better if you encase them in triple backticks (similar to triple double-quotes/block quotes in Python) or indent it by four spaces:

```suggestion
    listings/
    ├── brief
    │   ├── Capabilities4JdkSASL.txt
    │   ├── Capabilities4Mozilla-JSS.txt
    │   ├── Capabilities4SunJCE.txt
    │   ├── Capabilities4SunJSSE.txt
    │   ├── Capabilities4SunPKCS11.txt
    │   ├── Capabilities4SunRsaSign.txt
    │   └── Capabilities4SUN.txt
    └── verbose
        ├── Capabilities4JdkSASL.txt
        ├── Capabilities4Mozilla-JSS.txt
        ├── Capabilities4SunJCE.txt
        ├── Capabilities4SunJSSE.txt
        ├── Capabilities4SunPKCS11.txt
        ├── Capabilities4SunRsaSign.txt
        └── Capabilities4SUN.txt
```

This will turn it into a code block:

    listings/
    ├── brief
    │   ├── Capabilities4JdkSASL.txt
    │   ├── Capabilities4Mozilla-JSS.txt
    │   ├── Capabilities4SunJCE.txt
    │   ├── Capabilities4SunJSSE.txt
    │   ├── Capabilities4SunPKCS11.txt
    │   ├── Capabilities4SunRsaSign.txt
    │   └── Capabilities4SUN.txt
    └── verbose
        ├── Capabilities4JdkSASL.txt
        ├── Capabilities4Mozilla-JSS.txt
        ├── Capabilities4SunJCE.txt
        ├── Capabilities4SunJSSE.txt
        ├── Capabilities4SunPKCS11.txt
        ├── Capabilities4SunRsaSign.txt
        └── Capabilities4SUN.txt